test/test_limits.cpp

```diff
@@ -64,6 +56,13 @@ int main()
     static_assert(test_value(std::numeric_limits<decimal128>::denorm_min(), "1.0000000000000000000000000000000000000000e-6176"));
     static_assert(test_value(std::numeric_limits<decimal128_fast>::denorm_min(), "1.0000000000000000000000000000000000000000e-6143"));
 
+    static_assert(test_value(std::numeric_limits<decimal32>::max() + std::numeric_limits<decimal32>::lowest(), "0.0000000000000000000000000000000000000000e+00"));
```

Comment:
nit:  add same comment(// Lowest + max) as in other preprocessor branch to keep it consistent